src/main/java/lotto/util/ValidationUtils.java

```diff
@@ -0,0 +1,73 @@
+package lotto.util;
+
+import lotto.message.InputErrorMessage;
+
+import java.util.Set;
+
+public class ValidationUtils {
+
+    public static void validateBuyAmount(String buyInput) {
+        if (buyInput == null || buyInput.trim().isEmpty()) {
```

Comment:
이 부분 이렇게 바꿀 수도 있을 것 같아요!
```suggestion
        if (buyInput == null || buyInput.isBlank() ) {
```